src/main.js

```diff
@@ -1 +1,205 @@
+console.log(numbers);
+
+// Use the slice method to create a new array containing the last three elements of the original array.
+let startIndex = numbers.length - 3;
+let lastThreeElements = numbers.slice(startIndex);
+console.log(lastThreeElements);
+
+// Assignment 5: Array Sorting
+// Task:
+// Create an array of names.
+let names = ["Charlie", "Eve", "Alice", "Bob", "David"];
+
+// Sort the array in alphabetical order.
+names.sort();
+console.log(names);
+
+// Sort the array in reverse alphabetical order.
+names.reverse();
+console.log(names);
+
+// Assignment 6: Finding Elements in an Array
+// Task:
+// Create an array of numbers.
+let numbers2 = [10, 20, 30, 40, 50];
+
+// Use indexOf to find the position of a specific number.
+console.log(numbers2.indexOf(30));
+
+// Use includes to check if a specific number is present in the array.
+console.log(numbers2.includes(30));
+
+// Use findIndex to find the index of the first element greater than a certain value.
+let index = numbers2.findIndex(num => num > 25);
+console.log(index);
+
+// use for loop to find the index of the first element greater than a certain value.
+let value = 40;
+for (let i = 0; i < numbers2.length; i++) {
+    if (value < numbers2[i]) {
+        console.log(`index of the number which is greater then 40 is ${i}`);
+    }
+}
+
+// Assignment 7: Multi-dimensional Arrays
+// Task:
+
+// Create a 2D array representing a grid.
+let grid = [
+    [1, 2, 3],
+    [4, 5, 6],
+    [7, 8, 9]
+];
+
+// Access a specific element in the grid.
+console.log(grid[0][2]);
+
+// Modify an element in the grid.
+grid[2][2] = 4;
+console.log(grid[2][2]);
+
+// Assignment 8: Array Destructuring
+// Task:
+// Create an array with 4 elements.
+let array3 = ["apple", 42, true, "banana"];
+
+// Use destructuring to extract the first and second elements into variables.
+let [first, second] = array3;
+console.log(`first ${first}; second ${second}`);
+
+// Swap the values of two variables using array destructuring.
+[first, second] = [second, first];
+console.log(`first ${first}; second ${second}`);
+
+// Assignment 9: Combining Arrays
+// Task:
+// Create two arrays of numbers.
+let array5 = [1, 2, 3, 4, 5];
+let array6 = [10, 20, 30, 40, 50];
+
+// Combine the two arrays into one using the concat method.
+console.log(array5.concat(array6));
+
+// Create a new array by combining the two arrays using the spread operator.
+let combinedArray = [...array5, ...array6];
+console.log(combinedArray);
+
+// Assignment 10: Reversing and Sorting Arrays
+// Task:
+// Create an array of numbers.
+let numbers7 = [10, 5, 30, 2, 15];
+
+// Reverse the array.
+numbers7.reverse();
+console.log(numbers7);
+
+// Sort the array in descending order
+let numbers7Sort = [];
+let numLenght = numbers7.length;
+let firstN = numbers7[0];
+
+for (let i = 0; i < numLenght; i++) {
+
+    for (let i = 0; i < numLenght; i++) {
+        if (numbers7[i] > firstN) {
+            firstN = numbers7[i];   
+        }
+    }
```

Comment:
აქაც `i` არ გამოიყენო რამე სხვა ცვლადი ჯობს, მაგალითად `j`
```suggestion
    for (let j = 0; j < numLenght; j++) {
        if (numbers7[j] > firstN) {
            firstN = numbers7[j];   
        }
    }
```